### DISCUSSION TITLE
Add Screen Layout Selection In Screen Creation Modal

### DISCUSSION THREAD

**Discussion by brocococonut** [Budibase/budibase#6407] [Ideas]
As the title says, an option to pre-select the default layout for a new screen (be it a CRUDor blank screen) should be added to the relevant modal. This will make it so we don't have to manually update heaps of screens' layout options - which can get extremely arduous when you've added CRUD screens for all your tables in BudiBase.

**Comment by melohagan** [comment]
This is a brilliant idea!

Do you have examples of default layouts you would like?

I think I have seen this suggestion before, but I couldn't find a link to share.

**Reply by brocococonut** [reply]
I was actually thinking a little differently than adding to the default layouts. I was more meaning showing those layouts and also any custom ones in the menu when you create a new screen. 
![image](https://user-images.githubusercontent.com/6368283/174786062-d1e6caef-1280-486c-bebb-74feb09a3f07.png)
![image](https://user-images.githubusercontent.com/6368283/174787002-4622bd54-fc51-42c4-ba50-de61a45d701e.png)

I went ahead and just changed the wording and default in the dropdown to illustrate what I meant.

It's mainly so you don't constantly have to go to each and every screen and change the layout to the desired one. If you were to create CRUD screens for 5 tables, then that's 15 screens you have to change the layout for

**Comment by melohagan** [comment]
Ah @brocococonut I understand now. The layouts themselves will actually be completely removed with the new UI that's coming out next month. 

All of the configuration will be confined to the screen settings.
